sdk/provisioning/Azure.Provisioning/tests/BicepValues/BicepValueTests.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using NUnit.Framework;
+
+namespace Azure.Provisioning.Tests.BicepValues;
+
+public class BicepValueTests
+{
+    [TestCaseSource(nameof(LiteralBicepValueTestCases))]
+    public string ValidateLiteralBicepValue(BicepValue bicepValue)
+    {
+        return bicepValue.Compile().ToString();
```

Comment:
Nit - You can just do `.ToString()` because `BicepValue<T>.ToString()` will implicitly call `.Compile()` for you.